docs/source/api.rst

```diff
@@ -159,6 +159,18 @@ Chalice
         entire lambda function name.  This parameter is optional.  If it is
         not provided, the name of the python function will be used.
 
+   .. method:: lambda_function(name=None)
+
+      Create a pure lambda function that's not connected to anything.
+
+      See :doc:`purelambda` for more information.
+
+      :param name: The name of the function to use.  This name is combined
```

Comment:
I'm hesitant to do that because it's an implementation details and can (and has) changed a few times.  If I do document it I would have to say not to rely on this behavior.